playbooks/scaler/scaler.yaml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Scaling compute nodes
+  hosts: undercloud
+  become: yes
+  become_user: stack
+  gather_facts: no
+  vars:
+      up_limit: 2
+      down_limit: 1
+  tasks:
+      - name: Create backup copy of overcloud deply script
+        shell: "cp -pv  ~/overcloud_deploy.sh ~/overcloud_deploy.sh.bak"
+
+      - name: change list of nodes
+        lineinfile:
+            dest: "~/overcloud_deploy.sh"
+            regexp: '^                       --{{ scale_service }}-scale {{ up_limit }} \\'
```

Comment:
I think the "extra space" is not needed here since your regex search and replace a string rather than the whole line